pkg/server/plugin/upstreamauthority/spire/spire.go

```diff
@@ -28,18 +28,18 @@ func BuiltIn() catalog.Plugin {
 	return catalog.MakePlugin(pluginName, upstreamauthority.PluginServer(New()))
 }
 
-type spirePlugin struct {
+type SpirePlugin struct {
```

Comment:
why the rename? if we're going to rename, we should go with `Plugin` to be consistent.